src/komodo_globals.h

```diff
@@ -47,6 +47,9 @@ const int32_t nDecemberHardforkHeight = 1670000;   //December 2019 hardfork
 const uint32_t nS4Timestamp = 1592146800; //dPoW Season 4 2020 hardfork Sunday, June 14th, 2020 03:00:00 PM UTC
 const int32_t nS4HardforkHeight = 1922000;   //dPoW Season 4 2020 hardfork Sunday, June 14th, 2020 
 
+const uint32_t nS5Timestamp = 1623682800;  //dPoW Season 5 Monday, June 14th, 2021 (03:00:00 PM UTC)
+const int32_t nS5HardforkHeight = 2437300;  //dPoW Season 5 Monday, June 14th, 2021
```

Comment:
2437300 - 2384484 = 52816
52816/1440 = ~36.6 days from now

Probably ok, but likely to happen earlier than the comment states

Reply:
Using `height_in_future = last_block_number + ((future_time - time_now) / block_time)`
2384540 + ((1623682800 - 1620522112)/60) = 2437218.133333333

Can update just before merge to get closer value if needed, seems this will shift depending on block production rate in the interval.

Reply:
> 2437300 - 2384484 = 52816
> 52816/1440 = ~36.6 days from now
> 
> Probably ok, but likely to happen earlier than the comment states

![image](https://user-images.githubusercontent.com/21057518/117557617-642a0800-b03a-11eb-8c00-5fa3d6d44560.png)